### ISSUE TITLE
make running an assessment easier in a11y-self-validator

### ISSUE THREAD

**Issue by karanbirsingh** [microsoft/accessibility-insights-web#3205]
[a11y-self-validator](https://github.com/microsoft/accessibility-insights-web/blob/master/src/common/a11y-self-validator.ts) is sometimes used to self-check our UI for accessibility issues. It would be nice to add simple visualizations to highlight / log assessment instances to the console as well when requested. This would simplify self-assessments.

We might also use the save/load assessment behavior to help with this - maintaining a running assessment file or snapshot for each view & triaging the deltas (new instances) found per feature.

**Comment by pownkel** [comment]
Note that the notes in the [A11y pass template](https://microsoft.sharepoint.com/teams/DD_VSCS/CandC/Keros/_layouts/OneNote.aspx?id=%2Fteams%2FDD_VSCS%2FCandC%2FKeros%2FDocuments%2FKeros%20Engineering&wd=target%28A11y%20Assessments.one%7CE3B2B901-4E9C-44FC-91D5-2D42841155ED%2FTemplate%7C01765BFA-2A55-41B1-861E-0E53E667F76F%2F%29%20onenote:https://microsoft.sharepoint.com/teams/DD_VSCS/CandC/Keros/Documents/Keros%20Engineering/A11y%20Assessments.one#Template&section-id={E3B2B901-4E9C-44FC-91D5-2D42841155ED}&page-id={01765BFA-2A55-41B1-861E-0E53E667F76F}&end) should be updated once this is complete

**Comment by DaveTryon** [comment]
After reviewing the full list of issues in our backlog, we have concluded that we will not be addressing this issue.
